docs/reference/github-intro.md

```diff
@@ -0,0 +1,8 @@
+# Using Git, GitHub and GitHub Desktop (version control)
```

Comment:
This doc feels like in between an explainer and a reference document - not sure if we want to split up the explainer part and reference documents just being a collection of links to external docs?

Reply:
> my approach is thinking about efficient knowledge transfer rather than always lining up pedantically with the Diátaxis framework.

I think that's where we ended up having two systems - the pathway and the diataxis arrangement of docs. efficient knowledge transfer should ideally be punted off to arranging documents in order for that (hence my suggestion of having two separate docs). But I'm not too fussed at this point, there's a lot of docs that break this rule atm, guess that's the next step of reiterating.

Reply:
I would change the title of the document by removing the word "using" to avoid the expectation this could be a "how to". Otherwise fine to be a single reference doc imo.